### DISCUSSION TITLE
How to make the players can not ride the pokemob

### DISCUSSION THREAD

**Discussion by Star-Jirachi** [Pokecube-Development/Pokecube-Issues-and-Wiki#1107] [Q&A] [answered]
I already modified some configurations, but my server's players can still ride the pokemob :(

**Comment by Thutmose** [comment, marked as answer]
you could either use permissions, or just make the ridden mobs really really slow.

for the slow ones, change the ridden speed values in the server config file.

that config file is in `world/serverconfig/`, and is the `pokecube-server.toml` file

**Reply by Star-Jirachi** [reply]
ohhhhhhhh
thx :)
